src/scalacheck/org/scalacheck/commands/Commands.scala

```diff
@@ -77,7 +77,7 @@ trait Commands {
   def destroySut(sut: Sut): Unit
 
   /** The precondition for the initial state, when no commands yet have
-   *  run. This is used by ScalaCheck when command sequences are shrunk
+   *  run. This is used by ScalaCheck when command sequences are shrinked
```

Comment:
`shrunken`! Let's run that tool with `-locale US`, too! https://github.com/client9/misspell#locale

Reply:
```suggestion
   *  run. This is used by ScalaCheck when command sequences are shrunken
```

Reply:
```suggestion
   *  run. This is used by ScalaCheck when command sequences are shrunk
```

* 153,000 results https://www.google.com/search?q="are+shrunk"
* 59,500 results https://www.google.com/search?q="are+shrunken"
* 13,100 results https://www.google.com/search?q="are+shrinked"

Reply:
@dwijnand you're right - I didn't actually read the rest of the sentence

Reply:
Apologies, the `!`s were my way of signalling that I was being jokey, not that I was being emphatic or dogmatic. It's a bigger issue for ambiguous "they are drunk" and "they are drunken". Since the doc refers to the first state of a sequence that was shrunk, it could say "shrunken head" of the sequence.